stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterRegistry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.network;
+
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.GuardedBy;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AsyncPrettyPrinterRegistry {
+
+  @GuardedBy("this")
```

Comment:
Err, it seems to me that it's guarded by an instance that you lost (`new HashMap<>()`).  `@GuardedBy` is intending to communicate the exact monitor lock used to ensure thread safety.  In this case, you can't specify that and are not explicitly enforcing the monitor locking yourself so the statement is invalid.